lib/std/debug.zig

```diff
@@ -1252,6 +1210,492 @@ pub const DwarfInfo = struct {
     pub fn readString(self: *DwarfInfo) ![]u8 {
         return readStringRaw(self.allocator(), self.dwarf_in_stream);
     }
+
+    /// This function works in freestanding mode.
+    /// fn printLineFromFile(out_stream: var, line_info: LineInfo) !void
+    pub fn printSourceAtAddressDwarf(
+        self: *DwarfInfo,
```

Comment:
This PR is making a breaking change:

`std.debug.printSourceAtAddressDwarf` -> `std.debug.DwarfInfo.printSourceAtAddressDwarf`

I'm OK with making breaking changes here, but let's make sure these renames all make sense. In this case `Dwarf`  in the function name is redundant, and should be removed.

Would you mind putting the list of renames in the commit message or PR description (like above) so that it's easy to see a high level view of the new identifier paths?

Reply:
That's the only one! All the rest are not `pub`.